tracing/providers/jaeger/src/main/java/io/helidon/tracing/providers/jaeger/JaegerTracerBuilder.java

```diff
@@ -296,6 +305,13 @@ public JaegerTracerBuilder config(Config config) {
 
         config.get("global").asBoolean().ifPresent(this::registerGlobal);
 
+        config.get("span-processor-type").asString()
+                .ifPresent(it -> spanProcessorType(SpanProcessorType.valueOf(it.toUpperCase())));
+        config.get("exporter-timeout").asLong().ifPresent(it -> exporterTimeout(Duration.ofMillis(it)));
```

Comment:
This is still wrong. We definitely support `Duration` in config. This should be 
```
config.get("exporter-timeout").as(Duration.class).ifPresent(this::exporterTimeout);
```

I think the same works in Helidon 3 (and if not, we should add the mapper to default mappers).

Reply:
Ok, now I know this.